contrib/opencensus-ext-azure/opencensus/ext/azure/log_exporter/__init__.py

```diff
@@ -107,6 +108,16 @@ def stop(self, timeout=None):  # pragma: NO COVER
             return time.time() - start_time  # time taken to stop
 
 
+class SamplingFilter(logging.Filter):
```

Comment:
Is there any desire to support adaptive sampling for Application Insights as well? If so, it may be worth it to name this `FixedRateSamplingFilter` to be more specific.

Reply:
Currently in the roadmap there is no plan to add adaptive sampling. If this is ever added, it will be simple to iterate on this since the class is not exposed to customers.